src/constants.py

```diff
@@ -2,3 +2,9 @@ class SettingFileNames:
     GENERAL_FILE_NAME = "general.json"
     MONITORING_GROUPS_FILE_NAME = "monitoring_groups.json"
     RECIPIENTS_FILE_NAME = "recipients.json"
+
+
+class NotificationServiceConfig:
+    SES_SENDER = "my_email@soname.de"
+    ALERT_HEADER = "SDM [DEV] SAP Lean Glue Failure"
+    SMTP_SENDER = "my_email@soname.de"
```

Comment:
for ses &smtp sender - it's a good point, we need to put it somewhere, but more proper way to define it in general.json config.
Like:

    "delivery_methods": [
        {
            "name": "aws_ses",
            "delivery_method_type": "AWS_SES",
            "sender_email" : "salmon-no-reply@soname.de"
        },